index.html

```diff
@@ -4,22 +4,57 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Awesome Books</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="./index.css">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-9ndCyUaIbzAi2FUVXJi0CjmCapSmO7SnpJef0486qhLnuZ2cdeRhO02iuK6FUUVM" crossorigin="anonymous">
+<script src="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0/dist/js/bootstrap.bundle.min.js" integrity="sha384-geWF76RCwLtnZ8qwWowPQNguL3RmwHVBC9FhGdlKrxdiJJigb/j/68SIy3Te4Bkz" crossorigin="anonymous"></script>
 </head>
 <body>
-    <h1>Awesome Books</h1>
+
+    <header>
+        <nav class="header">
+    <h1 class="nav-head">Awesome Books</h1>
+            <ul>
+                <li id="toggleListButton"> list</li>
+                <li id="toggleAddButton"> Add new</li>
+                <li id="toggleContactButton"> contact</li>
+            </ul>
+        </nav>
+        <span id="currentTime"> </span>
+    </header>
+    <h1 id="h1">All Books</h1>
```

Comment:
- [ ] Good job so far, :clap: , However, Please ensure that you use only one `<h1>` in your entire html file, In HTML, the `<h1>` tag is used to define the main heading of a document. It represents the highest level of heading and carries the most importance in terms of semantic structure. It is generally considered best practice to have only one `<h1>` tag per HTML document. :slightly_smiling_face: